particle/particle/convert.py

```diff
@@ -163,7 +163,8 @@ def unmap(mapping):
     # This will override any negative values
     full.Latex.update(latex_series)
 
-    # These items are not very important - can be reconstructed from the PDG
+    # These items are not very important - can be reconstructed from the PDG ID
+    # TODO: maybe first check the consistency between what is read in and what the PDG ID provides (being maniac)?
```

Comment:
👍 This was done at one point when this was in a Jupyter notebook, but was removed once it matched. Should be added back in now that this is in a library file.

Reply:
This check will be left for post a release. There's already A LOT since 0.3.0 and a release will be necessary for the outstanding PRs in decaylanguage to pass.

Reply:
Yes, I was just agreeing with the comment. We should test before a release by using `python -m pip install git+https://github.com/scikit-hep/particle.git@master` so that we don't risk having an immediate 0.4.1, 0.4.2, etc.